### PR TITLE
[Load Balancing] Add caveat note about Pages origins

### PR DESCRIPTION
**Pull request by Erisa**

### CODE REVIEW

File: content/load-balancing/_partials/_pool-create.md [hunk 1]
```diff
@@ -23,6 +23,12 @@ You can create a pool within the [load balancer workflow](/load-balancing/load-b
     *   A [**Weight**](/load-balancing/understand-basics/traffic-steering/origin-level-steering/#weights)
     *   (Optional) A [hostname](/load-balancing/additional-options/override-http-host-headers/) by clicking **Add host header**
 
+{{<Aside type="note" header="Note">}}
+
+If your origin is pointing to [Cloudflare Pages](/pages/), you will need to fill in the host header field with the project URL for it to resolve correctly.
```
Comment (WalshyDev) [inline content/load-balancing/_partials/_pool-create.md]:
```suggestion
If your origin is pointing to [Cloudflare Pages](/pages/), you will need to fill in the host header field with the project (pages.dev) domain for it to resolve correctly.
```

Does this work with custom domains and other things?

Comment (Erisa) [reply content/load-balancing/_partials/_pool-create.md]: URL -> domain is already an edit I made in a followup commit.

Regarding custom domains, yes it seems to work: https://pageslb.erisa.party/
image

So probably not necessary to specify.

Comment (WalshyDev) [reply content/load-balancing/_partials/_pool-create.md]: Perfect, consider this a nevermind :)

**Review by WalshyDev — APPROVED** [review]

**Comment by Erisa** [comment]
For those of us who were curious if a note inside a partial inside a box works as intended, the answer is yes:

image

Preview URL since the bot doesnt understand partials: https://erisa-lb-pages-note.cloudflare-docs-7ou.pages.dev/load-balancing/pools/create-pool/#create-a-pool

**Review by RebeccaTamachiro — APPROVED** [review]

**Pull request merged**